source/ftrack_connect_pipeline/host/engine/asset_manager.py

```diff
@@ -622,14 +622,25 @@ def change_version(self, asset_info, options, plugin=None):
             asset_type_name = asset_entity['type']['name']
             version_id = asset_version_entity['id']
             location = asset_version_entity.session.pick_location()
-            component_path = None
+            component_path = component_id = None
```

Comment:
I don't think it's a good idea to = component_path to component_id, I know that this will work for this case as component_id is not set, but you never know how the code will be modified, so I think would be a better practice to just not mix these to variables that doesn't have a common value. 
So I suggest to do:
`component_path = None`
`component_id= None`

Reply:
Will change this, the code base needs to be reviewed as this statements exists in other places